src/routes/dbInit/import.spec.ts

```diff
@@ -0,0 +1,95 @@
+// tslint:disable-next-line
+import fetch from 'jest-fetch-mock';
```

Comment:
You don't need to disable TSLint for this next line, just change it to

```js
import { default as fetch } from 'jest-fetch-mock'
```

and you'll be okay.